src/examples/dropdowns/autocomplete/Size.tsx

```diff
@@ -0,0 +1,119 @@
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+import React, { useRef, useState, useEffect } from 'react';
+import debounce from 'lodash.debounce';
+import {
+  Item,
+  Hint,
+  Menu,
+  Label,
+  Field,
+  Dropdown,
+  Autocomplete
+} from '@zendeskgarden/react-dropdowns';
+import { Row, Col } from '@zendeskgarden/react-grid';
+
+const options = [
+  'Asparagus',
+  'Brussel sprouts',
+  'Cauliflower',
+  'Garlic',
+  'Jerusalem artichoke',
+  'Kale',
+  'Lettuce',
+  'Onion',
+  'Mushroom',
+  'Potato',
+  'Radish',
+  'Spinach',
+  'Tomato',
+  'Yam',
+  'Zucchini'
+];
+
+const Example = () => {
+  const [selectedItem, setSelectedItem] = useState(options[0]);
+  const [inputValue, setInputValue] = useState('');
+  const [matchingOptions, setMatchingOptions] = useState(options);
+
+  /**
+   * Debounce filtering
+   */
+  const filterMatchingOptionsRef = useRef(
+    debounce((value: string) => {
+      const matchedOptions = options.filter(
+        option => option.trim().toLowerCase().indexOf(value.trim().toLowerCase()) !== -1
+      );
+
+      setMatchingOptions(matchedOptions);
+    }, 300)
+  );
+
+  useEffect(() => {
+    filterMatchingOptionsRef.current(inputValue);
+  }, [inputValue]);
+
+  return (
+    <Row justifyContent="center">
```

Comment:
Add `alignItems="center"`. See https://5ef38236dc0966e836fbe6fa--zendeskgarden.netlify.app/components/select#size for comparison.